web/src/App.tsx

```diff
@@ -14,27 +12,25 @@ import NoMatch from "@/pages/NoMatch";
 import Settings from "@/pages/Settings";
 import UIPlayground from "./pages/UIPlayground";
 import Events from "./pages/Events";
-import { isDesktop } from "react-device-detect";
+import { isDesktop, isMobile } from "react-device-detect";
 import Statusbar from "./components/Statusbar";
+import Bottombar from "./components/navigation/Bottombar";
 
-function App() {
-  const [sheetOpen, setSheetOpen] = useState(false);
-
-  const toggleNavbar = () => {
-    setSheetOpen((prev) => !prev);
-  };
+const mobileInsets = "absolute left-0 top-2 right-0 bottom-0 bottom-16"
+const desktopInsets = "absolute left-16 top-16 top-2 right-0 bottom-8"
```

Comment:
How about `absolute left-0 md:left-16 top-2 right-0 bottom-16 md:bottom-8`? and just putting those on line 33?